lib/src/main/java/com/orange/ods/compose/component/button/OdsIconToggleButton.kt

```diff
@@ -70,11 +62,11 @@ fun OdsIconToggleButton(
             modifier = modifier,
             enabled = enabled
         ) {
-            Icon(
-                painter = if (checked) checkedPainter else uncheckedPainter,
-                contentDescription = iconContentDescription,
-                tint = iconButtonTintColor(displaySurface).enable(enabled = enabled)
-            )
+            if (checked) {
```

Comment:
This can be refactored to:

```kotlin
val icon = if (checked) checkedIcon else uncheckedIcon
icon.Content(enabled = enabled, displaySurface = displaySurface)
```